src/Weather.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class Weather extends Component {
+  state = {
+    temp: "",
+    pic: "",
+    description: "",
+  };
+
+  componentDidMount() {
+    fetch(
+      `http://api.weatherstack.com/current?access_key=${process.env.REACT_APP_WEATHER}&query=Toronto&units=m`
+    )
+      .then((res) => {
+        return res.json();
+      })
+      .then((data) =>
+        this.setState({
+          pic: data.current.weather_icons[0],
+          description: data.current.weather_descriptions[0],
+          temp: data.current.temperature,
+        })
+      );
+  }
+
+  render() {
+    return (
+      <div className="ui card" id="weather">
+        <div class="content">
+          <div class="image">
+            <img
+              className="ui mini right floated image"
+              src={this.state.pic}
```

Comment:
code looks good. just wanted to add destructuring the state variables at the top of the render method.